cypress/e2e/contact_dev_path.spec.cy.ts

```diff
@@ -0,0 +1,20 @@
+describe('The Home Page', () => {
+  it('Good Dev Local', () => {
+    cy.viewport(1920, 1080)
```

Comment:
I think we could set it in cypress.config.js as a global option for every test :)